test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -720,7 +720,7 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $result = ExecuteRequestWithOutFile -cmdletName "Invoke-WebRequest" -uri $uri
         $jsonContent = $result.Output | ConvertFrom-Json
         $jsonContent.headers.Host | Should Be $uri.Authority
-        $jsonContent.headers.'User-Agent' | Should Match "WindowsPowerShell"
+        $jsonContent.headers.'User-Agent' | Should Match "PowerShell"
```

Comment:
This match would work with `WindowsPowerShell` as well. This should be `^PowerShell`. 

In addition to that, it should be `MatchExactly`

Reply:
User-Agent can be `PowerShell/6.0.0-Beta.8` or `PowerShell/6.0.0`- so if we use  MatchExactly the pattern should be "^PowerShell/\d+\.\d+\.\d+.*"

Reply:
Makes sense.

Reply:
It's actually more like `Mozilla/5.0 (Windows NT; Microsoft Windows 10.0.15063 ; en-US) WindowsPowerShell/6.0.0` and it will be more dynamic "soon"

we could pull the exact one through reflection in `BeforeAll`. Honestly, I'm not even sure this is a good test. `Should Be $uri.Authority` should be sufficient, IMO.

Reply:
fixed.